test/functional/apps/dashboard_elements/links/links_navigation.ts

```diff
@@ -0,0 +1,229 @@
+  const security = getService('security');
+  const filterBar = getService('filterBar');
+  const testSubjects = getService('testSubjects');
+  const kibanaServer = getService('kibanaServer');
+  const dashboardAddPanel = getService('dashboardAddPanel');
+
+  const { dashboard, common, timePicker } = getPageObjects(['dashboard', 'common', 'timePicker']);
+
+  const FROM_TIME = 'Oct 22, 2018 @ 00:00:00.000';
+  const TO_TIME = 'Dec 3, 2018 @ 00:00:00.000';
+
+  describe('links panel', () => {
+    before(async () => {
+      await kibanaServer.savedObjects.cleanStandardList();
+      await security.testUser.setRoles([
+        'kibana_admin',
+        'kibana_sample_admin',
+        'test_logstash_reader',
+      ]);
+      await esArchiver.load('test/functional/fixtures/es_archiver/kibana_sample_data_flights');
+      await kibanaServer.importExport.load(
+        'test/functional/fixtures/kbn_archiver/dashboard/current/kibana'
+      );
+      await kibanaServer.importExport.load(
+        'test/functional/fixtures/kbn_archiver/kibana_sample_data_flights_index_pattern'
+      );
+      await kibanaServer.uiSettings.replace({
+        defaultIndex: '0bf35f60-3dc9-11e8-8660-4d65aa086b3c',
+      });
+      await common.setTime({
+        from: FROM_TIME,
+        to: TO_TIME,
+      });
+    });
+
+    after(async () => {
+      await kibanaServer.importExport.unload(
+        'test/functional/fixtures/kbn_archiver/kibana_sample_data_flights_index_pattern'
+      );
+      await esArchiver.unload('test/functional/fixtures/es_archiver/kibana_sample_data_flights');
+      await kibanaServer.uiSettings.unset('defaultIndex');
+      await common.unsetTime();
+      await security.testUser.restoreDefaults();
+    });
+
+    describe('embeddable panel', () => {
+      afterEach(async () => {
+        await dashboard.clickDiscardChanges();
+      });
+
+      it('adds links panel to top of dashboard', async () => {
+        await dashboard.loadSavedDashboard('links 003');
+        await dashboard.switchToEditMode();
+        await dashboardAddPanel.addEmbeddable('a few horizontal links', 'links');
+        const topPanelTitle = (await dashboard.getPanelTitles())[0];
+        expect(topPanelTitle).to.equal('a few horizontal links');
+      });
+    });
+
+    describe('dashboard links', () => {
+      afterEach(async () => {
+        // close any new tabs that were opened
+        const windowHandlers = await browser.getAllWindowHandles();
+        if (windowHandlers.length > 1) {
+          await browser.closeCurrentWindow();
+          await browser.switchToWindow(windowHandlers[0]);
+        }
+      });
+
+      it('should not be a clickable link for current dashboard', async () => {
+        await dashboard.loadSavedDashboard('links 002');
+        const listItem = await testSubjects.find('dashboardLink--link002');
+        const tagName = await listItem.getTagName();
+        expect(tagName).to.not.equal('a');
+      });
```

Comment:
Don't we have a component unit test for this already? 🤔

Reply:
Yes, we do. I will remove this.